cnew.py

```diff
@@ -100,37 +82,30 @@ def obtntime():
     if int(timesecs) < 10:  timesecs = "0" + timesecs
     return timehour + ":" + timemint + ":" + timesecs
 
-
-def randgene():
-    numb = 8
-    randstrg = "".join(secrets.choice("ABCDEF" + "0123456789") for i in range(numb))
-    return randstrg
-
-
-def chekroom(strg):
-    if len(strg) != 8:
-        return False
-    else:
-        try:
-            geee = int(strg, 16)
-            return strg.isupper()
-        except ValueError:
-            return False
-
-
-def chekpass(pswd):
+async def chekroom(username,chatroom,password,servaddr):
+    if len(chatroom)!=8:
+        return "False"
     try:
-        suit = Fernet(pswd)
-        return True
-    except:
-        return False
-
+        geee = int(chatroom,16)
+    except ValueError:
+        return "False"
+    global websocket
```

Comment:
Please abstain from using a global variable here as far as possible.

Reply:
Hi, as stated earlier, refraing from use of global variable, will create unwanted sockets that I have to keep track of when a new user enters and exists(closing them).  Still if this is wrong, please comment it.